compiler/parser/Lexer.x

```diff
@@ -2697,6 +2701,11 @@ lexer queueComments cont = do
     then queueComment (L (RealSrcSpan span) tok) >> lexer queueComments cont
     else cont (L (RealSrcSpan span) tok)
 
+-- Use this instead of 'lexer' in Parser.y to dump the tokens for debugging.
```

Comment:
I cherry picked this from upstream GHC. It’s super useful for debugging. You just need to swap out what is called from `Parser.y`.